v2/setup/mesosphere.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+source /etc/environment
+
+if [ -z $MESOS_SLAVE_IMAGE ]; then
```

Comment:
could we use this script to update the image? or do we use something else for that?